l10n_br_fiscal/models/ncm.py

```diff
@@ -16,6 +16,8 @@ class Ncm(models.Model):
     ]
     _description = "NCM"
 
+    active = fields.Boolean(default=True)
```

Comment:
Eu sugiro você adicionar esse atributo na classe ancestral l10n_br_fiscal.data.abstract, que é a classe base para todas as tabelas fiscais do l10n_br_fiscal, dessa forma esse campo vai ser herdado em todas as tabelas fiscais como (NCM, CFOP, CST e etc)

Reply:
Também seria interessante colocar na tela o campo ou o botão de ativo ou arquivado


```xml
<div class="oe_button_box" name="button_box">
    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
        <field name="active" widget="boolean_button" options="{&quot;terminology&quot;: &quot;archive&quot;}"/>
    </button>
</div>

Só é interessante verificar se o botão fica ativo em um registro somente leitura, porque os dados fiscais estão como somente leitura (o que está correto), só é permitido editar com a permissão "Fiscal Data Maintenance"
```

Reply:
Eu fiz mais ou menos o que você pediu @renatonlima , 

- Coloquei o active direto do `data.abstract`

- O botão fiz um pouco diferente:
Eu segui um padrão que estou vendo na 14. Que é só por o ribbon de arquivado quando for o caso, usando apenas o server action que vem por padrão quando se coloca o active no modelo. Adicionalmente, eu botei uma verificação para que esse action só possa ser usado por quem está no grupo `Fiscal Data Maintenance`.

Uma coisa que não gostei é que o active vai ficar para todos os modelos que herdam o abstract, porém não consigo por o ribbon para todos estes.